api/turing/models/routes.go

```diff
@@ -11,12 +11,14 @@ import (
 
 // Route maps onto the fiber.Component.
 type Route struct {
-	// Id of the route
+	// ID of the route
 	ID string `json:"id"`
 	// Type of the route
 	Type string `json:"type"`
 	// Endpoint to query
 	Endpoint string `json:"endpoint"`
+	// EndpointInfoURL (optional) holds URL to endpoint details page
+	EndpointInfoURL string `json:"endpoint_info_url"`
```

Comment:
I find the idea of an `EndpointInfoURL` kind of ad-hoc. Also, in the rare event that the Merlin app server is moved, all the DB records need to be updated.

Would it be better to create a `MerlinEndpoint` type that can be `nil` where N/A, and can otherwise store the model id? Then, for display, the UI can format the endpoint URL by calling the relevant Merlin APIs if needed. Wdyt?

Reply:
Another approach we can do is having a field like `annotations` with type `map[string]string` to add this addtional information to a Router object. 

The annotation approach gives more flexibility to add/remove the extra information in the future and makes it less coupled to other objects types. If we do not need to enforce having this extra field or having a relationship on that field, we can consider this approach.

Reply:
> I find the idea of an `EndpointInfoURL` kind of ad-hoc. Also, in the rare event that the Merlin app server is moved, all the DB records need to be updated.
> 
> Would it be better to create a `MerlinEndpoint` type that can be `nil` where N/A, and can otherwise store the model id? Then, for display, the UI can format the endpoint URL by calling the relevant Merlin APIs if needed. Wdyt?

I didn't want to introduce another point of coupling between Turing and Merlin, that's why decided to go with a generic `EndpointInfoURL`. But yes, maybe `Annotations`, suggested by David, would be a better idea.

Reply:
@davidheryanto @krithika369 I addressed your comments, please give it another look

Reply:
Ok looks good. Is there gonna be another PR to update the databasa schema and test the annotation field gets written to the database. Currently I believe this new annotation field will not be written to the database.

Reply:
`annotations` field, as well as any other fields of `Route` struct are being written into the database as json blob. The test for (de-)serialization of entire `Route` struct into json was updated correspondingly.